my-app/src/components/General/TaskCard.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import GearMenu from '../Home/GearMenu';
+import Card from 'react-bootstrap/Card';
+import { FaCalendarWeek } from 'react-icons/fa';
+import { FaPlay } from 'react-icons/fa';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import Button from 'react-bootstrap/Button';
+import ProgressBar from 'react-bootstrap/ProgressBar';
+import Container from 'react-bootstrap/Container';
+
+export default ({ title, taskGoal, description }) => {
+	const now = 85;
+
+	const progressInstance = <ProgressBar className="progressBar" animated now={now} label={`${now}min`} />;
+
+	const [card, setCard] = useState({
```

Comment:
Task card should be a stateless component

Reply:
oh okay thank you, got it!